client/src/components/CardItem/CardItem.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import {Typography, Box, styled} from "@mui/material";
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+
+const CardItemWrapper = styled("div")({
+    minWidth: "265px",
+    maxWidth: "265px",
+    height: "400px",
+    background: `url('../../assets/images/hotDeals/aerial-view-of-colorful-mixed-forest-shrouded-in-morning-fog-on-a-beautiful-autumn-day.jpg')`,
+    backgroundSize: "cover",
+    color: "white",
+    position: "relative",
+    cursor: "pointer",
+
+    "&::after": {
+        content: "''",
+        position: "absolute",
+        width: "100%",
+        height: "100%",
+        background: "rgba(0,0,0,0.3)",
+        left: "0",
+        top: "0",
+        transition: "opacity 0.5s",
+        opacity: "1",
+    },
+
+    "&:hover::after": {
+        opacity: "0",
+        cursor: "pointer",
+    }
+})
+
+const CardItemPrice = styled(Typography)(({theme}) => ({
+    backgroundColor: theme.palette.primary.main,
+    textAlign: "center",
+    width: "108px",
+    fontSize: "18px",
+    paddingTop: "19px",
+    paddingBottom: "19px",
+    fontWeight: "bold",
+    letterSpacing: "1px",
+    marginTop: "0",
+    position: "absolute",
+    zIndex: "1",
+    color: theme.palette.common.white,
+}))
+
+// eslint-disable-next-line react/jsx-props-no-spreading
+const CardItemHeader = styled((props) => <Typography variant="h3"{...props}/>)(({theme}) => ({
+    position: "absolute",
+    fontSize: "20px",
+    fontWeight: "bold",
+    marginTop: "265px",
+    paddingLeft: "25px",
+    zIndex: "2",
+    textTransform: "uppercase",
+    color: theme.palette.common.white,
+}))
+
+const CardItemLocation = styled('div')({
+    display: "flex",
+    paddingLeft: "25px",
+    paddingRight: "25px",
+    zIndex: "2",
+    position: "absolute",
+    width: "100%",
+    alignItems: "center",
+    marginTop: "350px",
+})
+
+const CardItemText = styled(Typography)(({theme}) => ({
+        fontSize: "14px",
```

Comment:
You do not need Font-size here as it`s already declarated in our theme. Please, delete it

Reply:
agree

Reply:
@Vladislav2498  please fix

Reply:
done